### ISSUE TITLE
Cannot find module which is added as dependency

### ISSUE THREAD

**Issue by ronaldruzicka** [codesandbox/codesandbox-client#4511]
# 🐛 bug report
## Description of the problem
I'm getting an error `Cannot find module 'react-hook-form'.ts(2307)` (happening also for other packages), even though I have them installed as dependencies"

## How has this issue affected you? What are you trying to accomplish?
I'm not getting correct Typescript type checking.

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | v3.7.0
| Browser          | Version 83.0.4103.116 (Official Build) (64-bit)
| Operating System | macOs Catalina 10.15.5

**Comment by CompuIves** [comment]
Heyo! The reason that it's happening here is because of the `include` field, as far I can see. If you empty the `includes` field in `tsconfig.json` the typing resolve correctly: http://localhost:3000/s/react-hook-form-typescript-06yji?file=/tsconfig.json:6-20

**Comment by ronaldruzicka** [comment]
Hi, thanks for the reply. Actually I checked `tsconfig.json` on our project and it was working with it. So I tried each option and `"moduleResolution": "node"` fixed the issue. But with `"moduleResolution": "classic"` it doesn't work. Don't really know why is that, but at least it's working now. Btw, it was also doing the same for `Yup` package, but I installed `@types/yup` and the error disappeared. But `react-hook-form` has already support for Typescript so it should work. 

Anyway `"moduleResolution": "node"` fixes it, so I close this issue.
